crucible-llvm-syntax/src/Lang/Crucible/LLVM/Syntax/TypeAlias.hs

```diff
@@ -0,0 +1,96 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Lang.Crucible.LLVM.Syntax.TypeAlias
```

Comment:
This deserves some module Haddocks, including motivation for why you'd want this sort of thing in the first place. This would be a good place to mention the fact that resolving type aliases requires knowledge of a full target triple (citing `x86_64LinuxTypes` and `aarch32LinuxTypes` as examples).

Reply:
Good point, done